R/PlotEMODResults.R

```diff
@@ -70,9 +70,10 @@ emodplot.prevalence <- function(data,
     mutate(Prevalence = case_when(Population == 0 ~ 0,
                                   Population > 0 ~ Infected / Population))
   y.lim.max <- min(max(data$Prevalence) * 1.2, 1)
-  emodplot.by_gender(data, date.start, date.end, 'Prevalence', title=title ) +
+  p <- emodplot.by_gender(data, date.start, date.end, 'Prevalence', title=title ) +
     scale_y_continuous(labels = scales::percent_format(accuracy = 1), breaks = seq(0, y.lim.max,0.05), limits=c(0,y.lim.max)) +
     ylab("HIV Prevalence (%)")
+  return(p)
```

Comment:
This is great. I don't think it changes behavior - an R function should return the output of the last line in the function. However, this is much easier to read.

Reply:
I was having trouble catching the plot object outside of the function before? Not sure what was going on, but this should fix it.